Sources/Player/OfflineEngine/Internal/Storage/OfflineEntry.swift

```diff
@@ -92,13 +95,15 @@ struct OfflineEntry: Codable {
 	}
 
 	init(
+		for mediaProduct: MediaProduct,
```

Comment:
Small suggestion, if the productId is all that is needed from the MediaProduct, then it's better not to rely on the full Object as an argument but then just the id.

Reply:
Fair! 👍 
I wanted to make it clear that the new `productId` was coming from `MediaProduct`, but it is not a good practice to pass all that extra context around.

Reply:
Made the change!